### PR TITLE
[Term Entry] PHP Arrays rsort() #2824

### PR DESCRIPTION
**Pull request by a-g-test**
### Description

- Added the rsort.md term for PHP
- Closes issue #2824

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/rsort/rsort.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+The output looks like this:
+
+0 = Mangos 
+1 = Kiwis 
+2 = Grapes 
+3 = Cherries
+
+
+## Codebyte Example
+
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/rsort/rsort.md:62]:
```suggestion
## Codebyte Example

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@a-g-test I did a first review and left a few comments.

File: content/php/concepts/arrays/terms/rsort/rsort.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+The **`rsort()`** sorts an indexed array in descending order.
+
+## Syntax
+
+```pseudo
+rsort(arrayToSort, sortType)
+```
+
+- `arrayToSort`: An array - required parameter, specifies the array to sort in rsort.
```
[the remark this answers is not in the document]
Comment (a-g-test) [reply content/php/concepts/arrays/terms/rsort/rsort.md]: This has been updated

File: content/php/concepts/arrays/terms/rsort/rsort.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+- `sortType`: A sort type - optional flag parameter, specifies how to compare the array elements.
+
+> **Note:** If two members compare as equal, they retain their original order. Prior to PHP 8.0.0, their relative order in the sorted array was undefined.
+
+###### Sorting type flags
```
[the remark this answers is not in the document]
Comment (a-g-test) [reply content/php/concepts/arrays/terms/rsort/rsort.md]: This has been updated

File: content/php/concepts/arrays/terms/rsort/rsort.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- **SORT_REGULAR** - compare items normally
+- **SORT_NUMERIC** - compare items numerically
+- **SORT_STRING**  - compare items as strings
+- **SORT_LOCALE_STRING** - compare items as strings, based on the current locale option (setlocale())
+- **SORT_NATURAL** - compare items as strings using "natural ordering" like natsort()
+- **SORT_FLAG_CASE** - sort strings case-insensitively with other flag options (SORT_STRING, SORT_NATURAL). 
+
+
+## Example
+
```
[the remark this answers is not in the document]
Comment (a-g-test) [reply content/php/concepts/arrays/terms/rsort/rsort.md:39]: This has been updated

File: content/php/concepts/arrays/terms/rsort/rsort.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+}
+?>
+```
+
+The output looks like this:
+
+0 = Mangos 
+1 = Kiwis 
+2 = Grapes 
+3 = Cherries
```
[the remark this answers is not in the document]
Comment (a-g-test) [reply content/php/concepts/arrays/terms/rsort/rsort.md:57]: This has been updated

**Review by SSwiniarski — APPROVED** [review]
@a-g-test Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@a-g-test @SSwiniarski I made a few small edits, but otherwise in good shape. Looks good to merge

**Comment by SSwiniarski** [comment]
@a-g-test Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/arrays/rsort

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**